### ISSUE TITLE
Free Courses links?

### ISSUE THREAD

**Issue by sadukie** [aspnet/Docs#5605]
## Requests for new Topics

Looking at https://docs.microsoft.com/en-us/aspnet/, I see that Free Courses links to asp.net/freecourses, which has some outdated links and does not appear to be part of this repo.  Are there plans to move that section to here?  (I would love to help with this if possible!)

**Comment by Rick-Anderson** [comment]
@jongalloway  who own's https://www.asp.net/freecourses ?

I think it would be better to remove the [Free courses](https://www.asp.net/freecourses) pane. Most of them are outdated. Both the ASP.NET tab and the ASP.NET Core tab use the same link.

cc @scottaddie  @rachelappel

**Comment by rachelappel** [comment]
@sadukie Hey Sarah! It's @LadyNaggaga who owns the free courses repo, so if you have further questions about the www.asp.net site, Maria's your gal. The docs team has been avoiding linking to courses so as not to appear to favor certain companies or individuals that produce videos. So unfortunately, we won't be able to move them. 

Anything from the docs.microsoft.com/aspnet or docs.microsoft.com/dotnet URLs my team covers, so ping any of us with questions: @Rick-Anderson, @scottaddie  and me on ASPNET, and @BillWagner, @mairaw on .NET.

**Comment by LadyNaggaga** [comment]
@Rick-Anderson  I agree we should probably the pane.  If in the future we see an opportunity to add it back then we can do so.  Thoughts ?

**Comment by terrimorton** [comment]
As @rachelappel said, it's @LadyNaggaga who owns that. I am the one who can make changes to it or remove it from the site. Considering it was promoted by Scott Hanselman and others, and the URLs probably exists in PowerPoints and whatnot, I would encourage a redirect or a commitment to update the content. Promising free courses and then leaving people at a 404 is not a good message.

**Comment by rowanmiller** [comment]
In Feb https://www.asp.net/freecourses got 17,000 users and 11,000 of them went on to one of the free courses so I think it's providing value. As we redesign the ASP.NET marketing site, we'll probably keep this content around - we've talked about adding it for .NET in general too.

I get not wanting to promote other companies on docs. On the marketing side we aren't too concerned about this. We vet the content to make sure we are only sending folks to quality courses.

We discussed removing the dates from the courses listed on the page, as most of them were published over a year ago but are still relevant, high quality courses.

**Comment by LadyNaggaga** [comment]
@rowanmiller that sounds like a good plan. I will start gathering newer content over the next couple of weeks.
